config-proto-converter/src/main/java/org/hypertrace/config/proto/converter/ConfigProtoConverter.java

```diff
@@ -0,0 +1,24 @@
+package org.hypertrace.config.proto.converter;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+import com.google.protobuf.Message;
+import com.google.protobuf.MessageOrBuilder;
+import com.google.protobuf.Value;
+import com.google.protobuf.util.JsonFormat;
+
+public class ConfigProtoConverter {
```

Comment:
This class converts from Proto to Json, should we add that to the class name itself ?

Reply:
Conversion from proto to json is just an intermediate step.
The 2 methods in the class are used to convert the `Value` proto object to another proto object(via `Message.Builder`) and vice-versa.

Reply:
Right - @GurtejSohi correctly summarizes this. Given that, do we have any name suggestions? I thought of naming it `ConfigProtoValueConverter` but felt that didn't lend any more clarity.